test/format/config_def.c

```diff
@@ -129,6 +129,13 @@ CONFIG configuration_list[] = {
   {"disk.mmap_all", "configure mmap operations (read and write)",
     C_BOOL, 5, 0, 0, V_GLOBAL_DISK_MMAP_ALL},
 
+/* Test format can only handle 32 tables so we use a maximum value of 32 here. */
+  {"file_manager.close_handle_minimum", "number of handles open before the file manager will look for handles to close",
+    0x0, 0, 20, 32, V_GLOBAL_FILE_MANAGER_CLOSE_HANDLE_MINIMUM},
```

Comment:
What does the three numbers here mean? 0, 20, 32?

Reply:
- 0 is the minimum value
- 20 is Maximum value randomly chosen
- 32 is the Maximum value explicitly set, this is from the ticket which says Test/format can only open up to 32 tables at once

Reply:
Thanks for the explanation. I thought the middle value is the default value but apparently I'm wrong. In that case, I agree we should set it as 0, 32, 32. Sorry for the confusion.

Reply:
@sonar3ddy @jiechenbo I think we should change it back to 0 32 32? What's your opinion?

Reply:
Yep lets do 32!